Creating_drug_data.R

```diff
@@ -125,7 +133,8 @@ if (powerTransformPhenotype) {
 }
```

Comment:
homogenizeData reporting error after this update. Checking for input and debugging now.